WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -56,19 +62,21 @@
     private int mSpinnerDrawableRight;
     private AppLog.T mTAG;
 
+    private boolean mToolbarVisibilityLock = false;
+    @LayoutRes int mSpinnerItemView = 0;
```

Comment:
I think these can be private.

Reply:
Fixed in ffc26c550114ce20ee2c6c72ac316958c72413b7